### PR TITLE
Get Overview Section to a good standard.

### PR DESCRIPTION
**Pull request by MiaAltieri**
This PR refers to the JIRA task: "Overview Section for MongoDB VM brought to "good standard"" 

Standard for Overview can be seen [here](https://docs.google.com/spreadsheets/d/15KlgOKWCw6s2Tdl_CETlDGdqY5wixtUW0OustEGouTs/edit?usp=sharing)

[Kubeflow](https://charmed-kubeflow.io/docs) was used as inspiration for this task

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
-MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB (vs 5.0) on physical or virtual machines as a single replica.
+The Charmed MongoDB Operator provides a data platform based on MongoDB. MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
 
+This operator charm deploys and operates MongoDB on physical or virtual machines. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling and configuration of MongDB in production in a reliable way.
```
Comment (deusebio) [inline README.md:10]:
```suggestion
This operator charm deploys and operates MongoDB on physical or virtual machines. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MongDB in production at scale in a reliable way.
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 
-## Description
+## Overview
 
-The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
 
-MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB (vs 5.0) on physical or virtual machines as a single replica.
+The Charmed MongoDB Operator provides a data platform based on MongoDB. MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
```
Comment (deusebio) [inline README.md:8]:
```suggestion
MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
```

**Review by deusebio — APPROVED** [review]
All good! Just some minors!

File: README.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
 
-MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB (vs 5.0) on physical or virtual machines as a single replica.
+MongoDB is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible experience for users; with easy to use data aggregation for data analytics. It is a distributed database, so vertical and horizontal scaling come naturally.
 
+This operator charm deploys and operates MongoDB on physical or virtual machines. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MongDB in production at scale in a reliable way.
```
Comment (delgod) [inline README.md:10]:
```suggestion
This operator charm deploys and operates MongoDB on physical or virtual machines. It offers features such as replication, TLS, password rotation, and easy to use integration with applications. The Charmed MongoDB Operator meets the need of deploying MongoDB in a structured and consistent manner while allowing the user flexibility in configuration. It simplifies deployment, scaling, configuration and management of MongoDB in production at scale in a reliable way.
```

**Review by delgod — APPROVED** [review]

**Pull request merged**